ui/app/utils/tensorzero.server.ts

```diff
@@ -60,26 +66,29 @@ export async function addHumanFeedback(formData: FormData) {
   } else if (!datapointId && !evaluatorInferenceId) {
     // Do nothing
   } else {
-    throw new ServerRequestError(
+    throw new TensorZeroServerError.InputValidation(
       "Either both or neither of datapointId and evaluatorInferenceId should be provided",
-      400,
     );
   }
   if ((episodeId && inferenceId) || (!episodeId && !inferenceId)) {
-    throw new ServerRequestError(
+    throw new TensorZeroServerError.InputValidation(
       "Exactly one of episodeId and inferenceId should be provided",
-      400,
     );
   }
-  const feedbackRequest = FeedbackRequestSchema.parse({
+  const feedbackRequest = FeedbackRequestSchema.safeParse({
     metric_name: metricName,
     value,
     episode_id: episodeId,
     inference_id: inferenceId,
     tags,
     internal: true,
   });
-  const response = await tensorZeroClient.feedback(feedbackRequest);
+  if (!feedbackRequest.success) {
```

Comment:
the error message here should include some useful info for the client and we should use it.